hotsos/defs/scenarios/landscape/bugs/lp2081056.yaml

```diff
@@ -0,0 +1,15 @@
+checks:
+  has_2081056:
+    input:
+      path: 'var/log/landscape-server/job-handler.log*'
```

Comment:
don't need the * as hotsos will take care of that for you

Reply:
Done